packs/data/feat-effects.db/effect-blade-ally.json

```
@@ -0,0 +1,201 @@
+                                "or": [
+                                    "class:champion:alignment:le",
+                                    "class:champion:alignment:ne",
+                                    "class:champion:alignment:ce"
+                                ]
+                            }
+                        ],
+                        "value": "fearsome"
+                    },
+                    {
+                        "label": "PF2E.WeaponPropertyRune.flaming.Name",
+                        "predicate": [
+                            "feat:radiant-blade-spirit",
+                            "divine-ally:blade"
+                        ],
+                        "value": "flaming"
+                    },
+                    {
+                        "label": "PF2E.WeaponPropertyRune.ghostTouch.Name",
+                        "predicate": [
+                            "divine-ally:blade",
+                            {
+                                "or": [
+                                    "class:champion:alignment:lg",
+                                    "class:champion:alignment:ng",
+                                    "class:champion:alignment:cg"
+                                ]
+                            }
+                        ],
+                        "value": "ghost-touch"
+                    },
+                    {
+                        "label": "PF2E.WeaponPropertyRune.holy.Name",
+                        "predicate": [
+                            "feat:radiant-blade-spirit",
+                            "divine-ally:blade",
+                            {
+                                "or": [
+                                    "class:champion:alignment:lg",
+                                    "class:champion:alignment:ng",
+                                    "class:champion:alignment:cg"
+                                ]
+                            }
+                        ],
+                        "value": "holy"
+                    },
+                    {
+                        "label": "PF2E.WeaponPropertyRune.keen.Name",
+                        "predicate": [
+                            "feat:radiant-blade-master",
+                            "divine-ally:blade"
+                        ],
+                        "value": "keen"
+                    },
+                    {
+                        "label": "PF2E.WeaponPropertyRune.unholy.Name",
+                        "predicate": [
+                            "feat:radiant-blade-spirit",
+                            "divine-ally:blade",
+                            {
+                                "or": [
+                                    "class:champion:alignment:le",
+                                    "class:champion:alignment:ne",
+                                    "class:champion:alignment:ce"
+                                ]
+                            }
+                        ],
```

```suggestion
                        "predicate": [
                            "feat:radiant-blade-spirit",
                            "self:trait:evil"
                        ],
```
This should work instead. Don't need to check for the feature in every choice since if anything it should be a predicate on the whole thing but also if they don't have the feature why are they adding the effect?  Also just truncate to the evil trait on the actor since that's a given anyways.